src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier() {
+        this.random = new Random();
+    }
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Color randomColor = colors[random.nextInt(colors.length)];
+        return randomColor.name();
     }
```

Comment:
It's good practice to avoid creating a new array each time the method is called. Since the array of colors will not change, you can store it as a constant field in the class.